pkgs/by-name/go/golink/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "golink";
+  version = "unstable-2024-01-26";
```

Comment:
version must start with a digit

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Reply:
Updated. Thanks for the hint.